index.html

```diff
@@ -7,26 +7,33 @@
         <meta name="author" content="Austin">
         <link rel="stylesheet" type="text/css" href="styles.css"> 
         <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins"/>
-
     </head>
     <body>
         <header>
           <div class="logo-container">
-            <a href=""><h3 id="logo">My Logo</h3></a>
+            <a href="#"><h3 id="logo">My Logo</h3></a>
+          </div>
+          <div class="drop-down" id="navMenu">
+            <ul class="dropdown-menu">
+             <h1 aria-hidden="true" id="menu-close" onclick="closeMenu()">X</h1>
+             <a href="#Home"><li class="menu-link" id="nav-btn" onclick="closeMenu()">Portfolio</li> </a>
+             <a href="#About"><li class="menu-link" id="nav-btn" onclick="closeMenu()">About</li> </a>
+             <a href="#Contact"><li class="menu-link" id="nav-btn" onclick="closeMenu()">Contact</li> </a>
+            </ul>
```

Comment:
- [ ] Kindly use the correct close button logo by making it smaller and choosing the correct icon to match the given design.
- [ ] Please remove the link staying for the mobile menu options. you can use CSS property `text-decoration: none;`

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://github.com/Obimbo07/Portfolio/assets/68796232/964c1f20-e311-4d5e-86bc-3dedb82cb7b9)  | ![image](https://github.com/Obimbo07/Portfolio/assets/68796232/45f680ff-cb39-45d5-b7b6-d5235cd3bc4f)  |